src/module.c

```diff
@@ -12312,6 +12312,23 @@ int RM_GetLFU(RedisModuleKey *key, long long *lfu_freq) {
  * ## Miscellaneous APIs
  * -------------------------------------------------------------------------- */
 
+/**
+ * Returns the full module options flags mask, using the return value
+ * the module can check if a certain set of module options are supported
+ * by the redis server version in use.
+ * Example:
+ *
+ *        int supportedFlags = RM_GetModuleOptionsFlagsAll();
+ *        if (supportedFlags & REDISMODULE_OPTIONS_ALLOW_NESTED_KEYSPACE_NOTIFICATIONS) {
+ *              // REDISMODULE_OPTIONS_ALLOW_NESTED_KEYSPACE_NOTIFICATIONS is supported
+ *        } else{
+ *              // REDISMODULE_OPTIONS_ALLOW_NESTED_KEYSPACE_NOTIFICATIONS is not supported
+ *        }
+ */
+int RM_GetModuleOptionsFlagsAll() {
```

Comment:
the API is `RM_SetModuleOptions`, so this one should be `RM_GetModuleOptionsAll` IMHO

Reply:
Ok, will change it.